app/src/main/java/edu/wpi/first/shuffleboard/app/MainWindowController.java

```diff
@@ -327,6 +331,7 @@ private void setup(Plugin plugin) {
   private void tearDown(Plugin plugin) {
```

Comment:
Note that `tearDown` does not need `FxUtils.runOnFxThread` since only `setup` can be called from the launcher thread in `Shuffleboard.init`